rest/rest-sqs/src/main/scala/org/elasticmq/rest/sqs/QueueAttributesOps.scala

```diff
@@ -0,0 +1,197 @@
+package org.elasticmq.rest.sqs
+
+import akka.actor.ActorRef
+import akka.util.Timeout
+import org.elasticmq.actor.reply._
+import org.elasticmq.msg._
+import org.elasticmq.rest.sqs.Constants._
+import org.elasticmq.rest.sqs.model.RedrivePolicy
+import org.elasticmq.rest.sqs.model.RedrivePolicy.BackwardCompatibleRedrivePolicy
+import org.elasticmq.rest.sqs.model.RedrivePolicyJson._
+import org.elasticmq.util.Logging
+import org.elasticmq.{DeadLettersQueueData, MillisVisibilityTimeout, QueueData}
+import org.joda.time.Duration
+import spray.json.JsonParser.ParsingException
+import spray.json._
+
+import scala.async.Async.{async, await}
+import scala.concurrent.{ExecutionContext, Future}
+
+trait QueueAttributesOps extends AttributesModule {
```

Comment:
is this only a refactor from `QueueAttributesDirectives` or are there some changes?

Reply:
refactor only - I've extracted logic here to reuse it in statistics module